site/docs/about/glossary.mdx

```diff
@@ -5,42 +5,88 @@ sidebar:
+A Salt component is a reusable building block underpinning the Salt Design System. User interface components encapsulate specific functionality and visual style, enabling designers and developers to create consistent, accessible and efficient digital experiences. Salt components can be combined and customized to support a wide range of use cases across applications. Users of Salt can design and implement their own [patterns](#pattern) within their own scope. For a full list of components, refer to our [component documentation](/salt/components/index).
+
+## Layout
+
+Salt layouts are a special category of components designed to manage the overall structure and arrangement of user interface elements, rather than providing visual widgets themselves. Layouts are responsible for organizing how other components are positioned and flow within an application or page. Read the overview for details on types of [layouts](/salt/components/layouts/index).
+
+## External dependency
+
+An external dependency refers to a user interface component sourced from a widely adopted third-party library or vendor, rather than developed in-house as part of the Salt Design System. We choose these best-in-class solutions that reflect standard choices within the development community. Salt integrates these components into applications to deliver advanced functionality and specialized interactions. To maintain visual consistency and a seamless user experience, we apply Salt's brand-aligned styles using dedicated theme/component packages. This approach ensures third-party dependencies are treated as peer dependencies, allowing consumers to update them independently and avoid issues related to version lock-in. Examples include: Data grid, built on AG Grid for highly interactive tables; Chart, built on Highcharts for a wide range of chart types; and [Splitter](/salt/components/splitter), built on React Resizable Panels to enable adjustable panel boundaries.
+
+## Stable
+
+A stable component in the Salt Design System is one that has undergone thorough testing and review. It is available for use in production environments after: achieving a known level of [accessibility](#accessibility); aligning with Salt’s brand guidelines; and confirming adherence to established business requirements. Such components are well-documented, supported and available in the @salt-ds/core package. Once marked as stable, they follow [semantic versioning](#versioning), meaning any breaking changes will be communicated in advance.
+
+## Beta
+
+The pre-release version of Salt elements in testing may contain bugs, incomplete features or performance problems. The "beta" tag denotes components that are under active development that may not function as expected. Feedback from the beta version helps identify any concerns before the "stable" release, so you should exercise caution. You can find beta components in the @salt-ds/lab package.
```

Comment:
Should we use the term 'element'? What do we mean by this? We don't have a definition for it. Should we just say 'version of a Salt component or pattern'?